app/middleware/locationValidator.js

```diff
@@ -30,4 +31,20 @@ function validateLocation(req, res, next) {
   }
 }
 
+function renderNoResultsPage(req, res) {
+  /* eslint-disable no-param-reassign */
+  res.locals.nearbyServices = [];
```

Comment:
I don't think there is any need to set either `nearbyServices` or `openServices` to empty arrays.